testutil/base.go

```diff
@@ -11,6 +11,12 @@ import (
 	dtypes "github.com/ovrclk/akash/x/deployment/types"
 )
 
+func init() {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount(CoinDenom, CoinDenom)
+	config.Seal()
```

Comment:
Let's see how this does.  The race detector may complain about it at some point.

Reply:
Race detector is happy so far.